inputs/simple_inner.json

```diff
@@ -1,6 +1,13 @@
 {
   "prebuild_plugins": [
     {
+      "args": {
+        "key": "is_autorebuild",
+        "value": "true"
+      },
```

Comment:
and we'll also need to provide `url`, `verify_ssl`, and `use_auth`, like with the other OSv3-using plugins.